_layouts/default.html

```diff
@@ -23,24 +23,25 @@
         <div class="site">
           <div class="header">
             <h1 class="title"><a href="{{ site.url }}/">{{ site.name }}</a></h1>
-            <a class="extra" href="{{ site.url }}/">Home</a>
-	    <a class="extra" href="{{ site.url }}/about/">About</a>
-	    <a class="extra" href="{{ site.url }}/topics/">Topics</a>
-	    <a class="extra" href="{{ site.url }}/code-of-conduct/">Code of Conduct</a>
-	    <a class="extra" href="{{ site.url }}/contact/">Contact</a>
+            <a class="extra" href="{{ site.url }}/about/">About</a>
+            <a class="extra" href="{{ site.url }}/topics/">Topics</a>
+            <a class="extra" href="{{ site.url }}/policies-procedures/">Policies/Procedures</a>
+<a class="extra" href="{{ site.url }}/contact/">Contact</a>
```

Comment:
Indenting based on hierarchy would have them correctly indented in my source file.
For most of the elements, I was just fixing the number of spaces on the indent as it wasn't correct.